index.js

```diff
@@ -0,0 +1,149 @@
+const alexa = require('ask-sdk-core')
+const skillBuilder = alexa.SkillBuilders.custom()
+const axios = require('axios')
+
+async function getDataForMovieByTitle(title, key) {
+    const apikey = process.env.IMDB_API_KEY
+    console.log(apikey)
+    const url = `http://www.omdbapi.com/?apikey=${apikey}&t=${title}`
+    const response = await axios.get(url)
+    return response.data[key]
+}
+
+const LaunchRequestHandler = {
+    canHandle(handlerInput) {
+        return handlerInput.requestEnvelope.request.type === 'LaunchRequest'
+    },
+    handle(handlerInput) {
+        const speakOutput = 'Welcome to Movie plot'
```

Comment:
It's a good idea to give the user guidance on how to use the skill in the launch request, typically if they are launching instead of going right into an intent it is because they don't know how to use the skill yet.